src/skirmishbunker/bunkerFloorCuts.py

```diff
@@ -11,7 +11,7 @@ def init_floor_cut(self):
     self.floor_cuts = None
 
 def make_floor_cuts(self):
-    height = self.wall_width + self.base_height + self.floor_tile_height
+    height = self.floor_thickness + self.base_height + self.floor_tile_height
```

Comment:
same deal, set a local variable default it to wall_thickness and swap it for floor_thickness when set